src/charm.lua

```diff
@@ -459,9 +459,17 @@ SMODS.Atlas {
     path = "BakeryCharms.png"
 }
 
+---Checks if the requirements are met for a Charm to be considered debuffed, in cases where the Charm cannot directly be accessed via a function variable.
+---@return boolean True if the Charm area is not present, is empty, or the Charm is debuffed.
+local function is_charm_debuffed_safe()
+    return not G.Bakery_charm_area or #G.Bakery_charm_area.cards == 0
+        or G.Bakery_charm_area.cards[1].debuff
+end
```

Comment:
This function should just be `return not G.GAME.Bakery_charm` (at which point it's inlined at the call site).

Reply:
Resolved in e626f1f